src/foam/mlang/mlang.js

```diff
@@ -2774,7 +2780,7 @@ foam.CLASS({
         List result = new ArrayList();
         if ( getProjectionWithClass() != null ) {
           for ( Object list: (ArrayList)getProjectionWithClass() ) {
-            Object[] obj1 = Arrays.copyOfRange((Object[])list, 1, ((Object[])list).length);
+            Object[] obj1 = Arrays.copyOfRange((Object[])list, 2, ((Object[])list).length);
```

Comment:
I'm not too keen of having magic 0's, 1's, and 2's all throughout the code. We should make these constants.

Reply:
fixed